test/raven.utils.js

```diff
@@ -108,6 +108,23 @@ describe('raven.utils', function() {
       parseStack([], callback);
       parseStack([{lol: 1}], callback);
     });
+
+    it('should extract context from last stack line', function(done){
```

Comment:
Well, actually the test does much more than that. Here it is what my point was when writing it:
1) A real js exception is caused on purpose inside the try block
2) It gets passed to parseStack along with a custom callback knowing that the callback gets invoked with an array of all the processed frames once the whole stack is parsed.
3) As the frames array gets reversed inside parseStack I know that the last frame corresponds to the file where the exception was generated and that the exception isn't inside native code.
4) I verify that the frame has properties: pre_context, context_line and post_context which is valid for non-internal stack lines only.
The test was failing without the change as inside parseStack the following was always resolving to ```true```: 
```js
isInternal = line.isNative() || (frame.filename[0] !== '/' && frame.filename[0] !== '.');
```
and the frame was never having pre_context, context_line and post_context